utils/validation.go

```diff
@@ -153,7 +153,7 @@ func LimitsMemoryValidation(memory *int) bool {
 	if memory == nil {
 		return true
 	}
-	if *memory < 128 || *memory > 512 {
+	if *memory < 128 || *memory > 2048 {
```

Comment:
This is a quick fix for now, but I think the validation should be remove or do a validation against backend

Reply:
agree